modules/SDL.lua

```diff
@@ -136,19 +136,16 @@ function SDL:StopSDL()
   self.autoStarted = false
   local status = self:CheckStatusSDL()
   if status == self.RUNNING then
-    local result = os.execute ('./tools/StopSDL.sh')
-    if result then
-      if config.storeFullSDLLogs == true then
-        sdl_logger.close()
-      end
-      return true
-    end
+    os.execute('./tools/StopSDL.sh')
   else
     local msg = "SDL had already stopped"
     xmlReporter.AddMessage("StopSDL", {["message"] = msg})
     print(console.setattr(msg, "cyan", 1))
-    return nil, msg
   end
+  if config.storeFullSDLLogs == true then
+    sdl_logger.close()
+  end
+  return true
```

Comment:
@dboltovskyi This function always returns `true` now. Maybe you should remove `return` statement.

Reply:
@aderiabin Yep, I've checked all test scripts where this function is used - there is no usage of return data. So this return statement can be removed

Reply:
@aderiabin Done in afc65aa